pkg/aws_config_server/assemble_config.go

```diff
@@ -33,7 +32,6 @@ func (a *ClientIDToAWSRoles) getRoles(ctx context.Context, masterRoles []string,
 		if err != nil {
 			return errors.Wrap(err, "Unable to get list of AWS Profiles")
 		}
-		logrus.Debugf("function: aws_config_server/assemble_config.go/getRoles(), accountList: %v", accountList)
```

Comment:
https://github.com/sirupsen/logrus#logging-method-name
in the future, if you want to display the call site of a log statement you can enable this (in the root command when the -v flag on)